landoapi/app.py

```diff
@@ -24,9 +29,17 @@ def create_app(version_path):
 
     # Get the Flask app being wrapped by the Connexion app.
     flask_app = app.app
+
     flask_app.config['VERSION_PATH'] = version_path
     log_config_change('VERSION_PATH', version_path)
 
+    version_info = json.load(open(version_path))
+    logger.info(version_info, 'app.version')
+
+    this_app_version = version_info['commit']
+    environment = version_info['build']
```

Comment:
As I understand it, the environment is one way to partition the set of exceptions thrown by Sentry.  Hostname is presumably another?  But if you have multiple hosts per environment, like in prod, then partitioning the pool of exceptions coming from all the different environments by hostname might be a pain.

I like the `ENVIRONMENT` suggestion.  I'd also like to know if Ops already has a way they do this.  @ckolos, are you aware of any standard-ish way to get the environment label a host lives under into the container data so Sentry can report on it?